Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+fybrik_python_logging = "==0.1.0"
+fybrik_python_vault = "==0.1.0"
```

Comment:
the latest python_vault version is 0.2.0

Reply:
Done